src/translations/en.json

```diff
@@ -262,7 +263,7 @@
     "challenge_period_progress": "There is a challenge period in progress with staging amount {staging_amount} {symbol}",
     "challenge_period_endtime": "The end time of this settlement is {endtime}. Once the challenge period is ended successfully, a dialog will be prompted for you to confirm the settlement in order to update the staged balance that is the available amount for withdrawing currency {symbol}.",
     "settling_payment": "Settling payment, waiting the transaction to be confirmed...",
-    "starting_payment_challenge": "Starting payment challenge, waiting the transaction to be confirmed...",
+    "starting_payment_challenge": "Waiting the settlement transaction to be confirmed...",
```

Comment:
_Waiting **for** the settlement transaction to be confirmed..._

Reply:
Thanks *for* pointing out :p